
{'date': 'Thu Apr 30 2020 20:34:28.430 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 10
  class oi()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Apr 30 2020 20:34:41.396 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 10
  class oi():
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Apr 30 2020 20:34:52.527 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 12
  self.SALA1.vai;
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Apr 30 2020 20:38:24.576 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 13
  self.sala1 = Cena(img=SALA1)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Apr 30 2020 20:39:11.431 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 13
  self.sala1 = Cena(img=SALA1)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Apr 30 2020 20:39:53.480 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 13
  self.sala1 = Cena(img="https://i.imgur.com/Z0jbt5J.png")
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Apr 30 2020 20:40:40.432 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 13
  self.sala1 = Cena(img="SALA")
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Apr 30 2020 20:41:41.303 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 13
  self.sala1=Cena(img=SALA)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Apr 30 2020 20:41:52.503 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 13
  self.sala1=Cena(img=SALA)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Apr 30 2020 20:43:42.430 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 13
  self.sala1=Cena(img=SALA)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Apr 30 2020 20:49:18.18 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 18
    oi()
  module <module> line 15
    self.textotalita3 = Texto (self.saladeaula, "Hoje a aula vai ser sobre uma importante escritora de literatura brasileira. Me ajude a encontrar o livro")
AttributeError: 'oi' object has no attribute 'saladeaula'
'''},
{'date': 'Thu Apr 30 2020 20:50:56.387 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 20
    oi()
  module <module> line 15
    self.quadro.entra(sala1)
NameError: name 'sala1' is not defined
'''},
{'date': 'Thu Apr 30 2020 20:51:11.727 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 20
    oi()
  module <module> line 15
    self.quadro.entra(SALA)
  module _spy.vitollino.main line 472
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'str' object has no attribute 'nome'
'''},
{'date': 'Thu Apr 30 2020 20:55:35.203 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 22
    oi()
  module <module> line 20
    self.textotalita3.foi = self.habilita
AttributeError: 'oi' object has no attribute 'habilita'
'''},
{'date': 'Thu Apr 30 2020 21:18:02.214 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 22
    oi()
  module <module> line 16
    self.quadro.entra(sala1)
NameError: name 'sala1' is not defined
'''},
{'date': 'Thu Apr 30 2020 21:20:49.46 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 20
    oi()
  module <module> line 17
    self.botao.entra(self.sala1)
  module _spy.vitollino.main line 472
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'Texto' object has no attribute 'nome'
'''},
{'date': 'Thu Apr 30 2020 21:22:18.881 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 21
    oi()
  module <module> line 14
    Texto(self.botao, "ols")
AttributeError: 'oi' object has no attribute 'botao'
'''},
{'date': 'Thu Apr 30 2020 21:24:03.55 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    oi()
  module <module> line 19
    self.falamenino= Texto(sala1, "Muito obrigado, Senhora Neide")
NameError: name 'sala1' is not defined
'''},
{'date': 'Mon May 04 2020 16:29:34.807 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 40
  self.pasta.entra(self
                                                                                                                                     ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon May 04 2020 16:29:53.187 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 45
    oi()
  module <module> line 36
    self.voltar1.vai=self.sala1.vai
AttributeError: 'oi' object has no attribute 'sala1'
'''},